exercises/concept/need-for-speed/.docs/hints.md

```diff
@@ -23,6 +23,7 @@
 
 - Assume the car is just starting the race
 - You need to calculate the maximum distance a car can drive with the current level of battery
+- The distance is a product of the car's speed and its remaining battery. The remaining battery can be calculated by `battery / batteryDrain`.
```

Comment:
I am sorry but this is still not correct.

The _remaining battery_ before the race is `battery`. The remaining battery after the race is `battery % batteryDrain`. None of those two interpretations of the term _remaining battery_ fit the expression `battery / batteryDrain`.

As mentioned before, `battery / batteryDrain` describes how many times one can call the "Drive" method on the car, that is why I suggested the name `numberOfDrives`. That multiplied with the speed gives the distance the case can drive in total.